tests/headingcommand.js

```diff
@@ -217,12 +204,12 @@ describe( 'HeadingCommand', () => {
 				);
```

Comment:
Please also change the above test because in this case:

```html
<heading1>foo[</heading1><heading2>bar</heading2><heading3>]baz</heading3>
```

The last block is not even returned by `getSelectedElements()` (it's a recent change) so this test checks less than it was supposed to.